plugins/inputs/mikrotik/parsers.go

```diff
@@ -0,0 +1,103 @@
+package mikrotik
+
+import (
+	"fmt"
+	"regexp"
+	"slices"
+	"strings"
+	"time"
+)
+
+func parse(data common) (points []parsedPoint, err error) {
+	var errorList []string
```

Comment:
Why not keep the errors as errors and maybe directly log them _or_ if you need to join them in the end using `errors.Join`?